packages/pluggableWidgets/time-series-chart-web/README.md

```diff
@@ -0,0 +1 @@
+Please see [TimeSeriesChart](https://docs.mendix.com/appstore/widgets/time-series-chart) in the Mendix documentation for details.
```

Comment:
```suggestion
Please see [TimeSeries](https://docs.mendix.com/appstore/widgets/time-series-chart) in the Mendix documentation for details.
```

Reply:
We might need to update the readme anyways before release as charts will be a module.